development/extensions/modification_to_extension.rst

```diff
@@ -672,18 +777,25 @@ And that's it. No file edits required for the template files aswell.
 Adding Events
 -------------
 
-You can also add events to your extensions php and template code. If you miss an event from the core, please post a topic into the `[3.x] Event Requests <https://area51.phpbb.com/phpBB/viewforum.php?f=111>`_-Forum and we will include it for the next release.
-We try to include a huge bunch of events by default, but surely we can not cover every place your MODs need to be covered.
+You can also add events to your extensions php and template code. If you miss an
+event from the core, please post a topic into the
+`[3.x] Event Requests <https://area51.phpbb.com/phpBB/viewforum.php?f=111>`_-Forum
+and we will include it for the next release. We try to include a huge bunch of
```

Comment:
Maybe:
There are already numerous events available. However, if your extension needs to make use of an event which is not yet In the core code you can request the event be added to the core by creating a ticket in the `phpBB Bug Tracker <https://tracker.phpbb.com/projects/PHPBB3>`_. and we will endeavour to include it in the next release.